jib-core/src/test/java/com/google/cloud/tools/jib/api/JibContainerBuilderTest.java

```diff
@@ -222,8 +222,7 @@ public void testToContainerBuildPlan_default() throws InvalidImageReferenceExcep
 
     ContainerBuildPlan buildPlan = containerBuilder.toContainerBuildPlan();
     Assert.assertEquals("base/image", buildPlan.getBaseImage());
-    Assert.assertEquals("amd64", buildPlan.getArchitectureHint());
-    Assert.assertEquals("linux", buildPlan.getOsHint());
+    ;
```

Comment:
Should remove the line container the stray `;`. However, I think we should rather test `getPlatforms()`.

Reply:
Done